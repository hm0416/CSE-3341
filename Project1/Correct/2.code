program
	int x;
begin
	ref y;
	y = new;
	y = 10;
	output y;
end