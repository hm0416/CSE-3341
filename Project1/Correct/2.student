program
	int x;
begin
	x=0-10;
end
