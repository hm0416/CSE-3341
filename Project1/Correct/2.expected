PROGRAM
INT
ID[X]
COMMA
ID[Y]
COMMA
ID[XY]
SEMICOLON
BEGIN
ID[X]
ASSIGN
CONST[0]
SEMICOLON
OUTPUT
ID[X]
SEMICOLON
END
